### PR TITLE
Build openpilot with webcam support in CI

### LINKED ISSUE #1898: Build openpilot with webcam support in CI
https://github.com/commaai/openpilot/tree/master/tools/webcam

### PR DESCRIPTION
**Pull request by adeebshihadeh**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -5,19 +5,20 @@ on:
 
 env:
   RUN: docker run --shm-size 1G --rm tmppilot /bin/sh -c
+  RUN_WEBCAM: docker run --shm-size 1G --rm tmppilotwebcam /bin/sh -c
```
Comment (adeebshihadeh) [inline .github/workflows/test.yaml:8]: this should just be in the webcam job

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -5,19 +5,20 @@ on:
...
   BUILD: |
-      # build the openpilot docker base image
       docker pull $(grep -ioP '(?<=^from)\s+\S+' Dockerfile.openpilot_base) || true
       docker pull docker.io/commaai/openpilot-base:latest || true
       docker build --cache-from docker.io/commaai/openpilot-base:latest -t commaai/openpilot-base:latest -f Dockerfile.openpilot_base .
 
-      # build the final CI image
       docker pull docker.io/commaai/openpilotci:latest || true
       docker build --cache-from docker.io/commaai/openpilotci:latest -t tmppilot -f Dockerfile.openpilotci .
+  BUILD_WEBCAM: |
```
Comment (adeebshihadeh) [inline .github/workflows/test.yaml:20]: same as above

File: .github/workflows/test.yaml [hunk 3]
```diff
@@ -90,13 +105,17 @@ jobs:
...
+      - name: Build webcam Docker image
+        run: eval "$BUILD_WEBCAM"
       - name: Push to dockerhub
         run: |
           docker login -u wmelching -p ${{ secrets.COMMA_DOCKERHUB_TOKEN}}
           docker tag commaai/openpilot-base:latest docker.io/commaai/openpilot-base:latest
           docker push docker.io/commaai/openpilot-base:latest
           docker tag tmppilot docker.io/commaai/openpilotci:latest
           docker push docker.io/commaai/openpilotci:latest
+          docker tag tmppilotwebcam docker.io/commaai/openpilotwebcamci:latest
```
Comment (adeebshihadeh) [inline .github/workflows/test.yaml:117]: push from the webcam job

**Review by ghost — COMMENTED** [review]
Oh I didn't know I could request a review from myself :)

**Pull request merged**